api/capi/src/nnstreamer-capi-util.c

```diff
@@ -79,7 +79,7 @@ ml_util_validate_tensor_info (const ml_tensor_info_s * info)
   if (!info)
     return ML_ERROR_INVALID_PARAMETER;
 
-  if (info->type < 0 || info->type >= ML_TENSOR_TYPE_UNKNOWN)
+  if (info->type == ML_TENSOR_TYPE_UNKNOWN)
```

Comment:
Is it safe to assume that ```type > ML_TENSOR_TYPE_UNKNOWN``` won't happen?

Reply:
I will change to switch-case to avoid this.